README.md

```diff
@@ -6,14 +6,68 @@ End-user apps in WSO2 Identity Server
 | :------------ |:------------- 
 | master      | [![Build Status](https://wso2.org/jenkins/view/Dashboard/job/docs/job/docs-is/badge/icon)](https://wso2.org/jenkins/view/Dashboard/job/docs/job/docs-is/) |
 
-## Prerequisite
+## Prerequisites
 
-To run the project on local, it requires [python](https://www.python.org/downloads/) & [mkdocs](https://www.mkdocs.org/)
+To run the project locally, it requires [python](https://www.python.org/downloads/) & [pip](https://pypi.org/project/pip/).
+
+### Install Python
+
+Check if you already has Python installed by running the following command.
+
+```bash
+$ python --version
+Python 2.7.10
+```
+
+If you see something like above, then `Python 2.7.10` is your default version.
```

Comment:
If you receive a response similar to the one shown above, `Python 2.7.10` is your default version.

Reply:
Fixed with `2b173a1`